runtime/lua/vim/_watch.lua

```diff
@@ -265,24 +262,23 @@ end
 --- @param opts vim._watch.Opts?
 --- @param callback vim._watch.Callback Callback for new events
 --- @return fun() cancel Stops the watcher
-function M.fswatch(path, opts, callback)
-  -- debounce isn't the same as latency but close enough
-  local latency = 0.5 -- seconds
-  if opts and opts.debounce then
-    latency = opts.debounce / 1000
-  end
-
+function M.inotify(path, opts, callback)
+  -- FIXME Follwo symlinks, yes or no?
```

Comment:
```suggestion
  -- FIXME Follow symlinks, yes or no?
```

Reply:
I forgot to remove it. I pass `--no-dereference` to not follow them, as it doesn't make sense.

Reply:
Fair, I was just pointing out the typo in the comment.